src/platform/plugins/shared/saved_objects_management/server/plugin.ts

```diff
@@ -19,28 +20,31 @@ export class SavedObjectsManagementPlugin
 {
   private readonly logger: Logger;
   private managementService$ = new Subject<SavedObjectsManagement>();
+  private capabilitiesService$ = new Subject<CapabilitiesStart>();
 
   constructor(private readonly context: PluginInitializerContext) {
     this.logger = this.context.logger.get();
   }
 
-  public setup({ http, capabilities }: CoreSetup) {
+  public async setup({ http, capabilities, getStartServices }: CoreSetup) {
```

Comment:
The use of async lifecycle methods in plugins is deprecated and highly discouraged. It looks like it's not necessary anyway.